docs/architecture/joins.txt

```diff
@@ -81,8 +83,12 @@ Joins in CrateDB
 ================
 
 CrateDB supports (a) CROSS JOIN, (b) INNER JOIN, (c) EQUI JOIN, (d) LEFT JOIN,
-(e) RIGHT JOIN and (f) FULL JOIN. To implement these, the nested loop join
-algorithm is implemented with a few optimizations.
+(e) RIGHT JOIN and (f) FULL JOIN. All of these join types are executed using
+the :ref:`nested loop join algorithm <joins_nested_loop>` along with some
+optimizations. :ref:`Equi Joins <joins_equi_join>` are executed using the
```

Comment:
maybe say "except the equi joins which are excuted using ..." ? 
(feels a bit weird to say " all are executed with NL" and then "equi joins are not")

Reply:
Sounds good. Just a note "Special optimizations, according to the specific use cases, are applied to improve execution performance."